nixos/modules/services/torrent/transmission.nix

```diff
@@ -508,9 +509,7 @@ in
         px ${cfg.settings.script-torrent-done-filename} -> &@{dirs},
       ''}
 
-      ${optionalString (cfg.webHome != null) ''
-        r ${cfg.webHome}/**,
-      ''}
+      r ${cfg.webHome}/**,
```

Comment:
Related to previous suggestion: this wouldn't need to be changed.

Reply:
Rollbacked this part